E/Exiv2/build_tarballs.jl

```diff
@@ -0,0 +1,58 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Exiv2"
+version = v"0.27.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/Exiv2/exiv2/archive/refs/tags/v$(version).tar.gz", "9fb2752c92f63c9853e0bef9768f21138eeac046280f40ded5f37d06a34880d9")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/exiv2-*
+
+if [[ "${target}" == *-linux-musl ]]; then
+    # Delete libexpat to prevent it from being picked up by mistake
+    rm /usr/lib/libexpat.so*
+fi
+
+mkdir build
+cd build/
+
+cmake .. \
+-DCMAKE_INSTALL_PREFIX=$prefix \
+-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+-DCMAKE_BUILD_TYPE=Release \
+-DEXIV2_BUILD_SAMPLES=OFF \
+-DIconv_INCLUDE_DIR=${includedir}
+
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libexiv2", :libexiv2),
+    ExecutableProduct("exiv2", :exiv2)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="Zlib_jll", uuid="83775a58-1f1d-513f-b197-d71354ab007a"))
+    Dependency(PackageSpec(name="Expat_jll", uuid="2e619515-83b5-522b-bb60-26c02a35a201"))
+    Dependency(PackageSpec(name="libpng_jll", uuid="b53b4c65-9356-5827-b1ea-8c7a1a84506f"))
+    Dependency(PackageSpec(name="Gettext_jll", uuid="78b55507-aeef-58d4-861c-77aaff3498b1"))
```

Comment:
Is gettext really needed at runtime?

Reply:
I don't think so? I added it following the [dependency list](https://github.com/exiv2/exiv2#24-dependencies) on the `README`. Looking at the top level `CMake` file again now, the flag that uses that is off by default, so we can probably get rid of it or comment it out?

Reply:
Gettext is an annoying dependency, if it can be avoided, much better